request.go

```diff
@@ -329,9 +328,8 @@ func (conn *Connection) UpdateTyped(space, index interface{}, key, ops interface
 }
 
 // CallTyped calls registered function.
-// It uses request code for Tarantool >= 1.7 if go-tarantool
-// was build with go_tarantool_call_17 tag.
-// Otherwise, uses request code for Tarantool 1.6.
+// It uses request code for Tarantool >= 1.7, so result is not converted
```

Comment:
"so result is not converted"

To be honest, I don't get what exact conversion  do you refer to. So there is a good chance that users will be confused too.

Reply:
I've removed "is not converted" and left a note only about a result type:
```
// Call17 calls registered Tarantool function.
// It uses request code for Tarantool >= 1.7, result is an array.

// CallTyped calls registered function.
// It uses request code for Tarantool >= 1.7, result is an array.

// Call16Typed calls registered function.
// It uses request code for Tarantool 1.6, result is an array of arrays.
// Deprecated since Tarantool 1.7.2.
```